packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -45,7 +45,7 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerFunction')
   })
 
-  test('factory#create context with reducers', () => {
+  test('factory#create context with reducers | using array reducer', () => {
```

Comment:
Can we remove the `|` but leave the stuff after?

`It should create a ReducerList with reducers using an array of reducers`?

Reply:
@victorsingh looks like you missed one?

Reply:
Yea, going to work on it now